make/agents/app/build.mk

```diff
@@ -0,0 +1,22 @@
+##@ Build
+
+AGENTSAPP_MAIN=./cmd/agents/app/main.go
+
+.PHONY: build
+build: fmt vet ## Build manager binary.
+	$(GO) build -o bin/agentapp ${AGENTSAPP_MAIN}
```

Comment:
I'm not a huge fan of having multiple definitions of the same rule, e.g. a `build` target for both primaza and agents.  Could we unify these somehow?

Reply:
as decided with the team, we are going to try this approach for now. In future we may roll back to a more make-ish approach or even introduce a different build automation tool